src/app/components/toolbar/index.html

```diff
@@ -20,6 +20,13 @@
     >
     Specifies a section to group items within the toolbar. The section displays items in a separate horizontal row.
     </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="sky-toolbar-view-actions"
+      isOptional="true"
+    >
+    Adds a section on the right side of the toolbar for items that substantially change or affect the view of the toolbar. This includes simple filters and view switchers. If the view section includes more than one item, simple filters appear on the left and view switchers appear on the right.
```

Comment:
LGTM, but a couple minor thoughts:

In the first sentence, do we need both "change" and "affect"? Would one of the other (or an alternative such as "alter") work just as well?

In the last sentence, it's probably just me, but I'd delete "view" before "section." The first time I read it, it confused me at first because I thought it sounded like we were talking about a different section somewhere else. I think deleting "view" would make this clearer, but that might just be me.

Also in the last sentence, we say simple filters go on the left and view switchers go on the right as if those are the only two choices. But the previous sentence makes it sound like those are just 2 possible items for this section. Are there other items that can appear in this section? And if so, how does their placement factor in to sections with multiple items?

Reply:
So for John's last question - we do not control the ordering here (or on the list version). Consumers give us the items to put in this section and we just render it. 

The usage guidelines do state that the order of common items for this section should be
1. Simple filters
2. Expand all/Collapse all buttons
3. View switcher

From a technical standpoint consumers could put whatever they want in this section and the ordering is totally up to them. This would go against the usage guidelines though.